docs/knowledge_base/tips/multithreading/index.md

```diff
@@ -0,0 +1,113 @@
+Taipy can display data that is generated in a separate thread. This is useful for displaying real-time data from a sensor or a simulator. For example displaying in a dashboard the information from sensors measuring air pollution around a city, or displaying CPU usage of a server.
```

Comment:
In order to facilitate reviews in GitHub, it is better to have lines with max 100 chars. 
At least for text content. It's not always feasible on code blocks, links, or html tags.

```suggestion
Taipy can display data that is generated in a separate thread. This is useful for displaying
real-time data from a sensor or a simulator. For example displaying in a dashboard the
information from sensors measuring air pollution around a city, or displaying CPU usage of
a server.
```